src/Select.js

```diff
@@ -122,7 +124,7 @@ const Select = React.createClass({
 			inputValue: '',
 			isFocused: false,
 			isLoading: false,
-			isOpen: false,
+			isOpen: this.props.isOpen !== null ? this.props.isOpen : false,
```

Comment:
I didn't know, so I made a [fiddle](https://jsfiddle.net/bobbyrenwick/a70Loco4/) that demonstrates what you suggested is the true behaviour!